### PR TITLE
[STOS-2] worker_poc

### PR DESCRIPTION
**Pull request by AdamNiesiobedzki**

### CODE REVIEW

File: messaging/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+worker/.mvn/wrapper/
+evaluator_mock/.mvn/wrapper/
```
Comment (tpalinski) [inline messaging/.gitignore]: To dac jako wildcard .mvn

File: messaging/evaluator_mock/src/main/java/com/stos/worker/WorkerApplication.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+			}  else if ("ep".equalsIgnoreCase(input)) {
+				evaluatorMock.sendGeneratedMessageRabbit();
+			} else if ("ec".equalsIgnoreCase(input)) {
+				evaluatorMock.receiveMessageRabbit();
+			}  else if ("evaluator_test".equalsIgnoreCase(input)){
+				evaluatorMock.evaluatorTest(100);
+			}
+		}
+	}
+}
```
Comment (tpalinski) [inline messaging/evaluator_mock/src/main/java/com/stos/worker/WorkerApplication.java]: Apka spada z rowerrka jak bedzie error xd

**Review by tpalinski — CHANGES_REQUESTED** [review]
wywalic mock evaluatora/gitignore i bedzie banglac

File: messaging/worker/mvnw
Comment (lukrzak) [inline messaging/worker/mvnw:1]: Delete mvnw and add it to the .gitignore file

**Review by tpalinski — COMMENTED** [review]
bedzie smigac

**Review by lukrzak — CHANGES_REQUESTED** [review]
Remove maven related files from worker directory

**Review by lukrzak — APPROVED** [review]
lgtm

**Review by tpalinski — APPROVED** [review]
git fixy

**Pull request merged**